gradio/utils.py

```diff
@@ -593,16 +592,19 @@ def is_update(val):
 
 
 def get_continuous_fn(fn: Callable, every: float) -> Callable:
-    def continuous_fn(*args):
+    # For Wasm-compatibility, we need to use asyncio.sleep() instead of time.sleep(),
+    # so we need to make the function async.
+    async def continuous_coro(*args):
         while True:
             output = fn(*args)
             if isinstance(output, GeneratorType):
-                yield from output
+                for item in output:
+                    yield item
```

Comment:
Why do we make this change?

Reply:
As commented here: https://github.com/gradio-app/gradio/pull/6099/files#diff-0dd827ecd796f1d09e48052f6f80c1c9cf519eb940595994c65744ca40b0ae6aR599

Reply:
I understand the change from `time.sleep()` to `await asyncio.sleep()` but why change:

`yield from output` to 

```py
for item in output:
     yield item
```

as far as I know, they are equivalent, but the latter is more verbose

Reply:
Because `yield from` is not allowed in an async function.

Pylance says:
![CleanShot 2023-11-07 at 23 39 17@2x](https://github.com/gradio-app/gradio/assets/3135397/d38fb277-d216-4a3a-b75d-5a1fcc796d6d)

or the Python interpreter says:
```
SyntaxError: 'yield from' inside async function
```

Reply:
Gotcha interesting